scripts/plotting/global_latlon_map.py

```diff
@@ -44,6 +47,43 @@ def global_latlon_map(adf):
     # - Take difference, calculate statistics
     # - make plot
 
+
+    #
+    # Use ADF api to get all necessary information
+    #
+    var_list = adfobj.read_config_var("diag_var_list")
+    basic_info_dict = adfobj.read_config_var("diag_basic_info")
+    plot_location = basic_info_dict["cam_diag_plot_loc"]
+    compare_obs = basic_info_dict["compare_obs"]
+
+    case_info_dict = adfobj.read_config_var("diag_cam_climo")
+    case_name = case_info_dict["cam_case_name"]
+    model_rgrid_loc = case_info_dict["cam_regrid_loc"]
+
+   # CAUTION: 
+   # "data" here refers to either obs or a baseline simulation, 
+   # Until those are both treated the same (via intake-esm or similar)
+   # we will do a simple check and switch options as needed:
+    if compare_obs: 
+        data_name = "obs"  # does not get used, is just here as a placemarker
+        data_list = adfobj.read_config_var("obs_type_list")  # Double caution!
+        data_loc = basic_info_dict['obs_climo_loc']
+
+    else:
+        data_info_dict = adfobj.read_config_var("diag_cam_baseline_climo") 
+        data_name = data_info_dict["cam_case_name"] # does not get used, is just here as a placemarker
+        data_list = [data_name] # gets used as just the name to search for climo files HAS TO BE LIST
+        data_loc = data_info_dict["cam_climo_loc"]
```

Comment:
All of this is already done below (likely brought in when you merged in main), so this whole code section can be deleted (except for possibly grabbing the `basic_info_dict` which is needed for `plot_type`).

Reply:
@nusbaume -- Yeah, I think something weird is going on with this. I think the merge did things that I didn't expect. I'll need to take a closer look, and maybe touch base with you later to make sure I merge things correctly.

Reply:
@brianpm I think in this case the issue can be remedied fairly easily (it looks like the merge put the conflicting code section just below yours).  I can go ahead and open a PR to the branch on your fork with the relevant fixes tonight (it will also give me a chance to test this new feature myself).